index.js

```diff
@@ -23,6 +26,11 @@ class ServerlessAppsyncPlugin {
         usage: 'DEPRECATED: Helps you update AppSync API',
         lifecycleEvents: ['update'],
       },
+      // TODO: this conflicts with serverless offline
```

Comment:
Curious what @sid88in thinks but my gut feeling is to name it `appsync-offline`.

Reply:
👍 sounds good!